user-data/a/UIChargeStatus

```diff
@@ -0,0 +1,12 @@
+{
+    "fid": 530,
+    "sb": 11,
+    "bl": 3,
+    "f": 1,
+    "so": 0,
+    "s": false,
+    "bo": true,
+    "v": 1,
+    "dl": false,
+    "cl": false
+}
```

Comment:
Just adding this comment here cause I don't know where better to add it.  There is a new analysisItem for the UIChargeStatus, but there isn't one for the Gear.

Do you want to include the Gear one as well?